src/DynamoPackages/PackageLoader.cs

```diff
@@ -148,10 +155,34 @@ private void InitPackageLoader(IEnumerable<string> packagesDirectories, string s
         {
             if (packagesDirectories == null)
                 throw new ArgumentNullException("packagesDirectories");
-            
-            this.packagesDirectories.Add(stdLibDirectory);
+
             this.packagesDirectories.AddRange(packagesDirectories);
 
+            // Setup standard library
+            var standardLibraryIndex = this.packagesDirectories.IndexOf(StandardLibraryToken);
+            if (standardLibraryIndex == -1)
+            {
+                // No standard library in list
+                // Add standard library first
+                this.packagesDirectories.Insert(0, stdLibDirectory);
+            }
+            else
+            {
+                // Replace token with runtime library location
+                this.packagesDirectories[standardLibraryIndex] = stdLibDirectory;
+            }
+
+            // Setup Default Package Directory
+            if (standardLibraryIndex == -1)
+            {
+                defaultPackagesDirectoryIndex = this.packagesDirectories.Count > 1 ? 1 : -1;
+            }
+            else
+            {
+                var safeIndex = this.packagesDirectories.Count > 1 ? 1 : -1;
+                defaultPackagesDirectoryIndex = standardLibraryIndex == 1 ? 0 : safeIndex;
```

Comment:
When will the `standardLibraryIndex `be something other than 0?

Reply:
as above, when a user moves it down using the manage node and package path UI.

Reply:
> @sm6srw - theres one AC that was included in the task that I don't see in your description:
> `if std.lib is disabled, the entry in the ui should appear disabled` - what do you think of this one? We could also leave this for later depending on if we expose those options to the UI as well.

Thanks, I missed this one. I will take a look.